packages/blade/src/components/Alert/__tests__/__snapshots__/Alert.native.test.tsx.snap

```diff
@@ -148,7 +148,7 @@ exports[`<Alert /> should render 1`] = `
               "fontSize": 12,
               "fontStyle": "normal",
               "fontWeight": "400",
-              "lineHeight": 20,
+              "lineHeight": 16,
```

Comment:
Breaking changes in native lineHeight? can you verify if all the components look okay on native once (along with all supported sizes/variants)? Would be quite hectic to check everything but I think we should check at least the components whose snapshots have been updated

Reply:
I've done the sanity check of all the components nothing seems to be breaking, just changes on the text sizes in few places.